src/components/Menudetails.js

```diff
@@ -0,0 +1,325 @@
+                        className="w-full py-3 ml-2 text-sm font-medium text-gray-900 dark:text-black"
+                      >
+                        Burgers
+                      </label>
+                    </div>
+                  </li>
+                  <li className="w-full border-b border-gray-200 rounded-t-lg dark:border-gray-600">
+                    <div className="flex items-center pl-3">
+                      <input
+                        id="angular-checkbox"
+                        type="checkbox"
+                        value=""
+                        className="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500"
+                      />
+                      <label
+                        htmlFor="angular-checkbox"
+                        className="w-full py-3 ml-2 text-sm font-medium text-gray-900 dark:text-black"
+                      >
+                        Pizza
+                      </label>
+                    </div>
+                  </li>
+                  <li className="w-full border-b border-gray-200 rounded-t-lg dark:border-gray-600">
+                    <div className="flex items-center pl-3">
+                      <input
+                        id="laravel-checkbox"
+                        type="checkbox"
+                        value=""
+                        className="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500"
+                      />
+                      <label
+                        htmlFor="laravel-checkbox"
+                        className="w-full py-3 ml-2 text-sm font-medium text-gray-900 dark:text-black"
+                      >
+                        Sauces
+                      </label>
+                    </div>
+                  </li>
+                </ul>
+              </div>
+            </div>
+
+            <Box sx={{ width: "100%" }}>
+              <Box>
+                <Tabs
+                  className="tabs"
+                  value={value}
+                  onChange={handleChange}
+                  aria-label="basic tabs example"
+                >
+                  <Tab className="tab" label="Menu" {...a11yProps(0)} />
+                  <Tab className="tab" label="Reviews" {...a11yProps(1)} />
+                  <Tab className="tab" label="Info" {...a11yProps(2)} />
+                </Tabs>
+              </Box>
+              <TabPanel value={value} index={0}>
+                <div className="col-12 bg-white p-4">
+                  <div className="orange-box "></div>
+                  <strong>Natural food Iems</strong>
+                  <div> </div>
+                  <div>Dosa</div>
+                </div>
```

Comment:
![image](https://user-images.githubusercontent.com/124079185/222811275-682848a6-3680-41c5-a91d-7ebfb6f6c75e.png)
Just asking, are we not following the wireframe or is there any change in the functionality as I can see price and a more buttons in the wireframe designed?